port/common/omrsysinfo_helpers.c

```diff
@@ -247,6 +269,7 @@ omrsysinfo_get_x86_description(struct OMRPortLibrary *portLibrary, OMRProcessorD
 	char vendor[12];
 	uint32_t familyCode = 0;
 	uint32_t processorSignature = 0;
+   uint32_t processorStepping = 0;
```

Comment:
Indentation issues here (mixed tabs and spaces)

Reply:
Fixed in latest push.  Thanks.